First hunk: 
  Include KDE4Workspace cmake magic so that KDE4WORKSPACE_KDECORATIONS_LIBS gets defined.

Second hunk:
  CMP0005 Preprocessor definition values are now escaped automatically.
  https://cmake.org/cmake/help/v3.0/policy/CMP0005.html

--- CMakeLists.txt.orig	2017-03-29 09:58:52 UTC
+++ CMakeLists.txt
@@ -3,6 +3,7 @@ project(KWIN_CRYSTAL)
 set(VERSION 2.2.1)
 
 find_package(KDE4 REQUIRED)
+find_package(KDE4Workspace REQUIRED)
 
 include (CPack)
 include (MacroLibrary)
@@ -42,7 +43,8 @@ include (KDE4Defaults)
 
 ########### global ###############
 
-add_definitions (${QT_DEFINITIONS} ${KDE4_DEFINITIONS} -DHAVE_CONFIG_H=1 -DVERSION=\\"${VERSION}\\")
+cmake_policy(SET CMP0005 NEW)
+add_definitions (${QT_DEFINITIONS} ${KDE4_DEFINITIONS} -DHAVE_CONFIG_H=1 -DVERSION="${VERSION}")
 include_directories (${KDE4_INCLUDES} ${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR})
 
 include_directories(
